Update SA setup screenshots, add light/dark mode images (#805)
autonomys/subspace-docs: docs/farming-&-staking/farming/space-acres/install.mdx

   - If you don't have one, click **Create wallet** to create a wallet.

3. **Path to Farm 0 and its size**:
   - This is where your **plots** (farming data) will be stored.
   - The size determines your probability of getting a reward on the Network. In most cases, larger plots (or larger farm size) will gain you more rewards.
   - The size could be set as a fixed size, e.g. 4TB, or 500GB, or as a percentage value of the available disk space.
   - If you prefer to have a few farms (e.g. you have some spare space on different SSD drives), you can click on **Add farm** to pledge some additional storage space.

   <ThemedImage
   alt="Configured setup screen"
   sources={{
      light: useBaseUrl('img/doc-imgs/space-acres/space-acres-setup-2-light.png'),
      dark: useBaseUrl('img/doc-imgs/space-acres/space-acres-setup-2-dark.png'),
   }}
   />

   :::tip
   The screen above is just an example of configured setup screen, make sure to use your own **Node path**, **Rewards address*, **Path to farm 0 and its size**, and **Fixed size / Disk space percentage** values
   :::

4. **Advanced configuration (optional)**:

>    - Toggle **Reduce plotting CPU load** (select this option if you're using the same PC for other tasks)

Reviewer: ```suggestion
   - Toggle **Reduce plotting CPU load** (select this option if you're using your farming PC for other tasks)
```
